odoo/tools/sql.py

```diff
@@ -121,21 +122,52 @@ def rename_column(cr, tablename, columnname1, columnname2):
 
 def convert_column(cr, tablename, columnname, columntype):
     """ Convert the column to the given type. """
+    using = f'"{columnname}"::{columntype}'
+    _convert_column(cr, tablename, columnname, columntype, using)
+
+def convert_column_translatable(cr, tablename, columnname, columntype):
+    """ Convert the column from/to a 'jsonb' translated field column. """
+    drop_index(cr, f"{tablename}_{columnname}_index", tablename)
+    if columntype == "jsonb":
+        using = f"""CASE WHEN "{columnname}" IS NOT NULL THEN jsonb_build_object('en_US', "{columnname}"::varchar) END"""
+    else:
+        using = f""""{columnname}"->>'en_US'"""
+    _convert_column(cr, tablename, columnname, columntype, using)
+
+def _convert_column(cr, tablename, columnname, columntype, using):
+    query = f'ALTER TABLE "{tablename}" ALTER COLUMN "{columnname}" TYPE {columntype} USING {using}'
```

Comment:
I got some errors with columns with a default value set.
```suggestion
    query = f'ALTER TABLE "{tablename}" ALTER COLUMN "{columnname}" DROP DEFAULT, ALTER COLUMN "{columnname}" TYPE {columntype} USING {using}'
```